vllm/attention/selector.py

```diff
@@ -88,13 +89,12 @@ def get_global_forced_attn_backend() -> Optional[_Backend]:
 
 @lru_cache(maxsize=None)
 def get_attn_backend(
-    num_heads: int,
     head_size: int,
-    num_kv_heads: int,
```

Comment:
Were these two variables never used before?

Reply:
Exactly. I'm guessing they were once used as constraints for certain attention backends that only supported some number of heads. For Mamba, I ran into problems when passing these in. I originally had something a bit more messy but noticed these weren't being used at all and opportunistically removed them.

@WoosukKwon could you take a look to see if you think this is OK, or if we anticipate these returning?